### ISSUE TITLE
Are the docs online?

### ISSUE THREAD

**Issue by rcooke** [dakrauth/django-swingtime#9]
Django Projects:
https://www.djangopackages.com/packages/p/django-swingtime/

Lists this as the Read The Docs page:
https://readthedocs.org/projects/django-swingtime/

But that is to a fork done years ago and never updated!

**Comment by dakrauth** [comment]
That is a build from someone else's fork. I've been trying to get the latest docs built on RTD, but for some reason they keep failing. Everything builds fine locally and there really isn't anything in the build log on the site to indicate the issue.

**Comment by rcooke** [comment]
I'll help if I can.  What is the RTFD url to your docs?

**Comment by dakrauth** [comment]
https://readthedocs.org/projects/swingtime/

I've published the docs on my own site as well: http://swingtime.nerdfog.com/ (and click the documentation button).

**Comment by rcooke** [comment]
Dude, these links should really be in your readme - it would have saved me a few grey hairs!

Is the nerdfog domain stable?  If it is, I will start by changing the Django Project entry for Swingtime to point to that instead of the Fiee outdate docs.

**Comment by dakrauth** [comment]
I never added the RTD links in the README doc for 2 reasons: swingtime pre-dated RTD, and I have no control over that domain or, apparently, how the docs are built (or not!). The nerdfog domain is just a stop gap while I investigate getting the current release and latest commits to build on RTD.

The docs and instructions for building them are included in this github repo, so you can always clone it and build locally, as well as run the demo project.

**Comment by rcooke** [comment]
True.  I see your points.  I would still advocate a paragraph about this in the readme, so people trying to evaluate your project for integration into other things don't wind up reading those outdated docs. 

If/when you move to a new domain, the Django Projects links can be revised. 

Thanks for the info!  I was "voluntold" to make an event calendar for a local Church. I figure if I have to do this once, I'm gonna make sure its available for others.  Your library is missing a couple of their requirements, so I'm trying to base on django-scheduler (not schedule), but I want to use your fantastic rule generator form.

**Comment by rcooke** [comment]
I'm going to assume that when you first created the RTFD project you choose "Sphinx" and not "MkDocs".  Please correct me if needed.

Can you push tags for all your releases?  It would be handy to try a build on older versions to see when/where it broke.  As per:

http://read-the-docs.readthedocs.org/en/latest/versions.html#how-we-envision-versions-working

**Comment by rcooke** [comment]
Have you tried erasing the build folder?

http://read-the-docs.readthedocs.org/en/latest/builds.html#deleting-a-stale-or-broken-build-environment
